elastic/beats
# Fix bug with override path in cgroups

## What does this PR do?

This is a fix for https://github.com/elastic/beats/issues/27598

In cases where `cgroupsHierarchyOverride` is set to `/`, and `ignoreRootCgroups` is set, we'll just completely bypass any cgroups reporting.

## Why is it important?

It's a bug that was breaking containers.

## Checklist



## How to test this PR locally

- Build metricbeat (or any other beat) for docker
- run on linux
- set `http.enabled` to true in the beat config
- From inside the docker container, do `curl localhost:5066/stats?pretty`
- Make sure cgroup metrics are reported.

## Related issues

- Fixes https://github.com/elastic/beats/issues/27598
